code/modules/mob/living/simple_animal/hostile/deer.dm

```diff
@@ -51,11 +51,11 @@ Have them evaluate mobs in their vicinity
 
 		if(H.m_intent == "run") //Don't run while stalking deer
 			spook_prob = 70
-		if(istype (H:wear_suit, /obj/item/clothing/suit/leather/deer))
+		if(H.get_item_by_slot(slot_wear_suit) == /obj/item/clothing/suit/leather/deer)
```

Comment:
Indeed, and is a type equal to an object?

Is a bridge equal to said bridge's construction plans?